src/RtcConfigure.tsx

```diff
@@ -165,7 +165,7 @@ const RtcConfigure: React.FC<Partial<RtcPropsInterface>> = (props) => {
     <Create dispatch={dispatch}>
       {(engineRef) => (
         <Join
-          precall={!props.callActive}
+          precall={!rtcProps.callActive}
```

Comment:
We need to change app-builder-core to use rtcProps instead of directly passing the prop to rtcConfigure for callActive